<launch>
    <node pkg="save_map_point" name="pub_map" type="pub_map_img.py" output="screen">
        <param name="map_png_path" value="$(find save_map_point)/map_point/Shiyitu2.png"/>
    </node>

    <!-- <node pkg="save_map_point" name="egvg_vis" type="egvg_vis.py" output="screen"/> -->
    
    <node name="rvizvisualisation" pkg="rviz" type="rviz" output="log" args="-d $(find plan_manage)/config/traj_egvg_vis.rviz" />

</launch>